roles/lifecycle/files/env.local

```diff
@@ -0,0 +1 @@
+APP_ENV=prod
```

Comment:
See my feedback in the block below. Maybe you need to add some to this template.